kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/pom.xml

```diff
@@ -16,23 +16,23 @@
     <dependencies>
```

Comment:
kie-server-controller-standalone version should be available from kie bom, this dependencyManagement section could be deleted.